### PR TITLE
OAS-3988 added domain restrictions

### PR DESCRIPTION
**Pull request by ansoboleva**

### CODE REVIEW

File: oasis/access-control.md [hunk 1]
```diff
@@ -421,12 +421,21 @@ Deployment Z of project DEF                  | ✓ | — | —
...
-To enhance security, you can limit which authentication providers are accepted for users trying to access an organization in Oasis.
+To enhance security, you can implement the following restrictions via [Oasisctl](oasisctl.html):
 
-To do this, you need to use [Oasisctl](oasisctl.html). The following commands are available:
+1. Limit allowed authentication providers.
+2. Specify an allowed domain list.
 
-- `get organization authentication providers` - allows you to see which authentication providers are enabled for accessing a specific organization
-- `update organization authentication providers` - allows you to update a list of authentication providers for an organization to which the authenticated user has access
+Using the first option, you can limit which **authentication providers** are accepted for users trying to access an organization in Oasis. The following commands are available:
```
Comment (ewoutp) [inline oasis/access-control.md:429]: ...are available to configure this option:

File: oasis/access-control.md [hunk 2]
```diff
@@ -421,12 +421,21 @@ Deployment Z of project DEF                  | ✓ | — | —
...
+
+- `oasisctl get organization authentication providers` - allows you to see which authentication providers are enabled for accessing a specific organization
+- `oasisctl update organization authentication providers` - allows you to update a list of authentication providers for an organization to which the authenticated user has access
   - `--enable-github` - if set, allow access from user accounts authenticated via Github
   - `--enable-google` - if set, allow access from user accounts authenticated via Google
-  - `--enable-username-password` - if set, allow access from user accounts authenticated via a username/password
\ No newline at end of file
+  - `--enable-username-password` - if set, allow access from user accounts authenticated via a username/password
+
+Using the second option, you can configure a **list of domains**, and only users with email addresses from the specified domains will be able to access an organization. The following commands are available:
```
Comment (ewoutp) [inline oasis/access-control.md:437]: ...are available to configure this option:

File: oasis/access-control.md [hunk 3]
```diff
@@ -421,12 +421,21 @@ Deployment Z of project DEF                  | ✓ | — | —
 
 ## Restricting access to organizations
 
-To enhance security, you can limit which authentication providers are accepted for users trying to access an organization in Oasis.
+To enhance security, you can implement the following restrictions via [Oasisctl](oasisctl.html):
 
-To do this, you need to use [Oasisctl](oasisctl.html). The following commands are available:
+1. Limit allowed authentication providers.
+2. Specify an allowed domain list.
```
Comment (ewoutp) [inline oasis/access-control.md:427]: Add note below these options:

Note that users who do not meet the restrictions will not be granted permissions for any resource in
the organization. These users can still be a member of the organization.

**Review by ewoutp — APPROVED** [review]
Few text improvements.
Then LGTM

File: oasis/access-control.md (hunk 3 above)
Comment (ansoboleva) [reply oasis/access-control.md:427]: Done.

File: oasis/access-control.md (hunk 1 above)
Comment (ansoboleva) [reply oasis/access-control.md]: Done.

File: oasis/access-control.md (hunk 2 above)
Comment (ansoboleva) [reply oasis/access-control.md]: Done.

**Pull request merged**